.gitignore

```diff
@@ -107,3 +107,8 @@ tags
 #
 .vscode
 .history
+
+#
+# Personal circuit scratch directory
+#
+circuits/
```

Comment:
This looks like an individual developer configuration, it probably shouldn't end up in the master branch.

Reply:
Pushed a change to remove this from the pull request